src/Runner.Worker/Handlers/ContainerActionHandler.cs

```diff
@@ -44,12 +44,9 @@ public async Task RunAsync(ActionRunStage stage)
             {
                 Data.Image = Data.Image.Substring("docker://".Length);
             }
-            else if (Data.Image.EndsWith("Dockerfile") || Data.Image.EndsWith("dockerfile"))
+            else if (IsDockerFile(Data.Image))
             {
-                // ensure docker file exist
                 var dockerFile = Path.Combine(ActionDirectory, Data.Image);
```

Comment:
Style mini NIT: Up to you, `IsDockerFile(Data.Image)` -> `TryParseDockerFile(Data.Image, out var dockerFile`) might be more idiomatic.